model/xobject.go

```diff
@@ -483,6 +487,16 @@ func (ximg *XObjectImage) SetImage(img *Image, cs PdfColorspace) error {
 	return nil
 }
 
+// GetParamsDict returns *core.PdfObjectDictionary with a set of basic image parameters.
+func (ximg *XObjectImage) GetParamsDict() *core.PdfObjectDictionary {
```

Comment:
No need to make this method public yet, as it's not used in other packages, as it was the case with the one in the `model.Image` struct.

Reply:
Done.